sdk/src/main/java/com/horizen/transaction/mainchain/BwtRequest.java

```diff
@@ -4,12 +4,13 @@
 import com.google.common.primitives.Ints;
 import com.horizen.block.MainchainTxBwtRequestCrosschainOutput;
 import com.horizen.box.WithdrawalRequestBox;
+import com.horizen.cryptolibprovider.FieldElementUtils;
 import com.horizen.utils.BytesUtils;
 import com.horizen.utils.Utils;
 
-import java.util.Arrays;
-
 public final class BwtRequest implements SidechainRelatedMainchainOutput<WithdrawalRequestBox> {
+    public static final int TRANSACTION_HASH_LENGTH = FieldElementUtils.fieldElementLength();
```

Comment:
1) `TRANSACTION_HASH_LENGTH = Utils.SHA256_LENGTH`
Currently both SHA256_LENGTH and fieldElementLength are equal.
2) `TRANSACTION_HASH_LENGTH` is defined in every `SidechainRelatedMainchainOutput` class. Let's put it once to `MainchainTransaction` or `CommonParams`, for example.